docs/notebooks/Supervised_WMD.ipynb

```diff
@@ -0,0 +1,1441 @@
+{
```

Comment:
<p>This should be shorter, and you can remove the authors' name and year (since the the goal is to incorporate in a documentation)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>